addon/utils/key-codes.js

```diff
@@ -71,9 +71,7 @@ const all = {
   y: 89,
   z: 90,
 
-  cmd: 91,
   meta: 91,
-  windowLeft: 91,
   metaRight: 92,
   cmdRight: 92,
```

Comment:
they don't map to `metaKey` on the event so I wanted to keep these in case someone wants to use them. we could maybe take them out later.